user_data/os_common.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 sudo bash -c 'echo "session required pam_limits.so" >> /etc/pam.d/common-session'
-sudo bash -c 'echo "*      soft    nofile      2000000"  >> /etc/security/limits.conf'
-sudo bash -c 'echo "*      hard    nofile      2000000"  >> /etc/security/limits.conf'
+sudo bash -c 'echo "*      soft    nofile      100000000"  >> /etc/security/limits.conf'
+sudo bash -c 'echo "*      hard    nofile      100000000"  >> /etc/security/limits.conf'
 
 echo "net.ipv4.tcp_tw_reuse=1" >>  /etc/sysctl.d/99-sysctl.conf
 echo "fs.nr_open=8000000" >>  /etc/sysctl.d/99-sysctl.conf
```

Comment:
Q: Should this be `1000000000` as well?

Reply:
I'd suggest we put this in a variable, so we can just use it on those occurrences.

Reply:
right, it should be some systemctl command to reload it.

Reply:
Now I tried and remember both `systemd` and `sysctl -p` refused to set the value I want.